Themes/default/Languages/es.ini

```diff
@@ -8,71 +11,73 @@ EventStageCounter=Fase %03i
 
 [ScreenTitleMenu]
 HelpText=&BACK; Salir &START; Seleccionar &MENUUP;&MENUDOWN; Mover
-Network OK=Red OK
+Network OK=Conexión OK
 Offline=Sin conexión
 Connected to %s=Conectado a %s
-CurrentGametype=Modo de juego actual: %s
-LifeDifficulty=Dificultad de vida: %s
-TimingDifficulty=Dificultad de ritmo: %s
-%i Songs (%i Groups), %i Courses=%i Songs (%i Groups), %i Courses
+CurrentGametype=Tipo de juego: %s
+LifeDifficulty=Dificultad vida: %s
+TimingDifficulty=Dificultad ritmo: %s
+%i Songs (%i Groups), %i Courses=%i canciones (%i grupos), %i cursos
 
 [ScreenTitleJoin]
-HelpText=Presiona &START; para Jugar
-HelpTextJoin=Presiona &START; para Jugar
-HelpTextWait=Insert Credits...
+HelpText=Pulsa &START; para jugar
+HelpTextJoin=Pulsa &START; para jugar
+HelpTextWait=Inserta créditos...
 EventMode=Modo evento
-JointPremiumMain=Joint Premium
-JointPremiumSecondary=Two players can play with one credit!
-DoublesPremiumMain=Doubles Premium
-DoublesPremiumSecondary=Play doubles for one credit!
+JointPremiumMain=Premium mixto
+JointPremiumSecondary=¡Con un crédito pueden participar dos jugadores!
+DoublesPremiumMain=Premium dobles
+DoublesPremiumSecondary=¡Con un crédito puedes jugar a dobles!
 
 [ScreenDemonstration]
 Demonstration=Demostración
 
 [ScreenQuickSetupOverview]
-Explanation=Quick Setup provides a easy way to set commonly-changed and useful preferences.
+Explanation=La configuración rápida te permite cambiar opciones comunes de forma sencilla.
 
 [ScreenProfileLoad]
-HelpText=... Cargando perfiles, espera ...
+HelpText=Cargando perfiles...
 
 [ScreenSelectProfile]
 %d Song Played=%d canción jugada
 %d Songs Played=%d canciones jugadas
-PressStart=Presiona\n&START;\npara unirte.
+PressStart=Presiona\n&START;\npara unirte
 HelpText=&MENUUP;&MENUDOWN; Cambiar perfil &START; Elegir/Unirse &BACK; Deseleccionar
 
 [ScreenSelectMaster]
-HelpText=&BACK; Salir &START; Seleccionar / Unirse &MENULEFT;&MENURIGHT; Mover
+HelpText=&BACK; Salir &START; Elegir/Unirse &MENULEFT;&MENURIGHT; Mover
 
 [ScreenSelectPlayMode]
-HelpText=&BACK; Salir &START; Seleccionar / Unirse &MENULEFT;&MENURIGHT; Mover
-EasyExplanation=Un modo para principiantes
-HardExplanation=Para expertos
-OldNormalExplanation=Ni muy fácil ni muy difícil
-NormalExplanation=Juega a todas tus canciones favoritas
-RaveExplanation=Lucha contra un amigo o un enemigo
-ExtendedExplanation=Varias canciones de seguido
-OniExplanation=Una auténtica prueba de habilidad
-EndlessExplanation=No se para, siempre continúa
+HelpText=&BACK; Salir &START; Elegir/Unirse &MENULEFT;&MENURIGHT; Mover
+EasyExplanation=Modalidad para principiantes
+HardExplanation=Solo para expertos
+OldNormalExplanation=Ni muy fácil, ni muy difícil
+NormalExplanation=¡Juega con tus canciones favoritas!
+RaveExplanation=Lucha contra un amigo o rival
+ExtendedExplanation=Varias canciones seguidas
+OniExplanation=Pon a prueba tu habilidad
+EndlessExplanation=A tope y sin parar
+NonstopExplanation=Carias canciones seguidas
```

Comment:
I'm doing a thorough proofreading with a friend and... yeah, some typos here and there, but will be fixed very, very soon, I promise! :dancer: